x-pack/plugins/monitoring/server/lib/metrics/classes/metric.ts

```diff
@@ -23,6 +23,7 @@ interface OptionalMetricOptions {
   metricAgg?: string;
   mbField?: string;
   type?: string;
+  isNotSupportedInInternalCollection?: boolean;
```

Comment:
This has some conceptual overlap with `mbField` for me.

It's probably fine for now, but I suspect over time we'll need to be more responsive to the data available rather than gating on internal/mb/whatever collection method.

On a related note, it doesn't look like we use this value outside of tests. I would have expected it to show up in the display code to show/hide graphs. Am I just missing it or was it intentionally left out?

Reply:
Opened https://github.com/elastic/kibana/issues/133258 as a point of discussion about the best way to do this in the future